pkg/synopsysctl/cmd_migrate.go

```diff
@@ -585,6 +590,29 @@ func migrateOpsSight(namespace string) error {
 	return nil
 }
 
+// migrateSize will ensure that the Size CRD exists as well as the default sizes
+func migrateSize(namespace string) error {
+	isClusterScoped := util.GetClusterScope(apiExtensionClient)
+	if _, err := util.GetCustomResourceDefinition(apiExtensionClient, util.SizeCRDName); err != nil {
+		crd, err := getCrdConfigs(namespace, isClusterScoped, []string{util.SizeCRDName})
+		if err != nil {
+			return err
+		}
+		if err := deployCrds(namespace, isClusterScoped, crd); err != nil {
+			return err
+		}
+	}
```

Comment:
Will need to do a wait here to ensure CRDs exist before creation of CRs.

Reply:
done